Makefile

```diff
@@ -109,3 +114,15 @@ u-boot-exit:
 
 clean:
 	-rm -rf u-boot opensbi linux *.bin *.elf *.cpio.gz
+
+
+module_board:
```

Comment:
What does this thing do? It's not obvious for someone without the background.

Could you add some comments on top of this rule just to explain what is going on? This will be helpful in the future to maintain the make script.